autoload/leaderf/python/leaderf/commandExpl.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import re
+from .utils import *
+from .explorer import *
+from .manager import *
+
+# :command result line
+# "!|  LeaderfBufTag     0      Leaderf<bang> bufTag"
+#      ^^^^^^^^^^^^^
+RE_USER_DEFINED_COMMAND = re.compile(r"^.{4}(\w+)")
+
+# index.txt line
+# "|:silent|	:sil[ent]	..."
+#    ^^^^^^
+RE_BUILT_IN_COMMAND = re.compile(r"^\|:([^|]+)\|")
+
+#*****************************************************
+# CommandExplorer
+#*****************************************************
+class CommandExplorer(Explorer):
+    def __init__(self):
+        self._content = []
+
+    def getContent(self, *args, **kwargs):
+        if self._content:
+            return self._content
+        else:
+            return self.getFreshContent()
+
+    def getFreshContent(self, *args, **kwargs):
+        result_list = []
+
+        # user-defined Ex commands
+        lfCmd("let tmp = @x")
+        lfCmd("redir @x")
+        lfCmd("silent command")
+
+        result = lfEval("@x")
+
+        lfCmd("let @x = tmp")
+        lfCmd("redir END")
```

Comment:
It's better to use execute() instead.
e.g.,
`result = lfEval("execute('command')")`

Reply:
That looks better.
Thanks.

I wrote the code referring to [historyExpl.py](https://github.com/Yggdroot/LeaderF/blob/b339f54e8cb0022557f5cc8e26a43f05aa06b37f/autoload/leaderf/python/leaderf/historyExpl.py#L24-L39).
Should `historyExpl.py` be changed to code that uses `lfEval()` as well?

Reply:
Let's keep it as it is.

Reply:
OK